src/triage/cli.py

```diff
@@ -252,7 +228,14 @@ def __init__(self, parser):
             "features across different cohorts"
         )
 
-        parser.set_defaults(validate=True, validate_only=False, materialize_fromobjs=True)
+        parser.add_argument(
+            "--show-timechop",
+            action="store_true",
+            default=False,
+            help="Visualize time chops (temporal cross-validation blocks')"
+        )
+
+        parser.set_defaults(validate=False, validate_only=False, materialize_fromobjs=True)
```

Comment:
...This was intentional? To change the `validate` default from `True` to `False`?